docs/dev/protobuf-features.md

```diff
@@ -1,24 +1,50 @@
+At least some of these steps will be automated over time, but the 
+process as is shouldn't be overly burdensome as long as the issue
+is reasonably scoped and the protobuf changes are designed carefully
+at the beginning of the issue.
 
-Suppose your feature is for issue number `123`. There are basically five steps to 
-set up development for the feature:
+# Setup
+
+Suppose your feature is for issue number `123`. Follow these steps to set up and 
+iterate development of the feature.
  1. Create a `0123-D-DoThings` branch off `develop` in the [Hedera protobufs](https://github.com/hashgraph/hedera-protobufs) 
     repository; make your changes to the _*.proto_ in this branch.
- 2. Create a `0123-M-DoThings` branch off `0883-M-V12ReleasePrep` in the [Hedera protobufs Java API](https://github.com/hashgraph/hedera-protobuf) 
-    repository; copy the _*.proto_ from your `0123-D-DoThings` branch in the Hedera protobufs repo.
- 3. Suppose the `version` element in the `master` _pom.xml_ above is `0.12.0-SNAPSHOT`. Then in 
-    the `0123-M-DoThings` branch, change this to `0.12.0-issue.123-SNAPSHOT`.
- 4. Make sure you have Sonatype OSS credentials in your _settings.xml_ as described [here](../release-automation.md). Then in 
-    the [Hedera protobufs Java API](https://github.com/hashgraph/hedera-protobuf) repository, run `mvn -Prelease deploy`. 
-    If this succeeds, you should see output like,
-    ``` ...  Uploading to ossrh: https://oss.sonatype.org/content/repositories/snapshots/com/hedera/hashgraph/hedera-protobuf-java-api/0.12.0-issue.123-SNAPSHOT/hedera-protobuf-java-api-0.12.0-issue.123-20210202.205133-4.jar ... ```
- 5. Finally, in your `0123-M-DoThings` branch in the main [Hedera Services](https://github.com/hashgraph/hedera-services) 
-    repository (that is, _this_ repository), :exclamation: **make certain** you have merged the `0883-M-UseProtoArtifact` branch :exclamation:, and then 
-    update the `hapi-proto.version` property to `0.12.0-issue.123-SNAPSHOT`.
-
-At this point you should be able to commit your changes and push to GitHub, and CircleCI flows should run successfully.
+ 2. Create a `0123-M-DoThings` branch off `master` in the [Hedera protobufs Java](https://github.com/hashgraph/hedera-protobufs-java) 
+    repository; sync your protobuf changes to this branch and use it to deploy 
+    snapshot versions of the `com.hedera.hashgraph:hedera-protobuf-java-api` artifact
+    to the OSSRH snapshot repository. 
+    - In this branch, change the `version` element in the _pom.xml_ to reflect your issue.
+      For example, if the working version is `0.12.0-SNAPSHOT`, change it to `0.12.0-issue.123-SNAPSHOT`.
+    - Ensure you have valid Sonatype OSS credentials in your _settings.xml_ as described [here](../release-automation.md). 
+    - Each time you change the protobuf in the Hedera protobufs repo and sync to the Hedera protobufs Java repo, 
+      run `mvn deploy` here to update the snapshot artifact for your feature.
```

Comment:
Can we add some details here on how to verify that our changes are deployed (uploaded) to the snapshot repository?

Reply:
If the build doesn't fail with `ERROR`, then they should be uploaded. 😁

Reply:
Ok, that works.